index.html

```diff
@@ -13,10 +13,70 @@
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>My Vlog</title>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <div class="header">
+
+<div class="topnav">
+  <a class="active1" href="#home">Home</a>
+  <a  class="active2" href="#contact">Contact</a>
+  <a class="active3"  href="#about">About me</a>
+</div>
+      <h2>Iran</h2>
```

Comment:
I feel like it would probably make more sense for this to be a h1, and then leave the h2 two for the titles of the articles. Then, when you use the :first-child pseudo-class, the first articule title would stand out.